todo.py

```diff
@@ -1,11 +1,31 @@
 import json
+import os
+from datetime import datetime
+
+def get_last_modified_timestamp(file_path):
+    try:
+        timestamp = os.path.getmtime(file_path)
+        return datetime.fromtimestamp(timestamp).strftime("%Y-%m-%d %H:%M:%S")
+    except FileNotFoundError:
+        return None
+    
+def load_tasks():
+    file_path = "tasks.json"
+    with open(file_path, "r") as file:
+        tasks = json.load(file)
```

Comment:
You should handle the exception if file is not read correctly
```suggestion
    with open(file_path, "r") as file:
        try:
            tasks = json.load(file)
        except json.JSONDecodeError as e:
            print(f"Error reading JSON from file: {e}")
        except Exception as e:
            print(f"An unexpected error occurred: {e}")
```